package.json

```diff
@@ -10,7 +10,7 @@
     "start": "ts-node src/index.ts",
     "prebuild": "rm -rf dist",
     "build": "tsc --outDir dist",
-    "postbuild": "npm run generate",
+    "prepublishOnly": "cd demo-app/client && npm i && npm run generate",
```

Comment:
this is the first time I am using the "prepublishOnly" tag. Please, when publishing the package, check it is working correctly.